packages/lib-classifier/src/store/WorkflowStepStore.js

```diff
@@ -59,14 +60,36 @@ const WorkflowStepStore = types
       addDisposer(self, workflowDisposer)
     }
 
+    function createClassificationObserver () {
+      const classificationDisposer = autorun(() => {
+        const classificationStore = getRoot(self).classifications
+        onAction(classificationStore, (call) => {
```

Comment:
Writing tests on a separate branch and MST threw this:
`Warning: Attaching onAction listeners to non root nodes is dangerous: No events will be emitted for actions initiated higher up in the tree.`

Sooooo, I suppose I gotta rethink what's going on here.